tests/PHPUnit/Fixtures/LatestStableInstall.php

```diff
@@ -57,16 +59,19 @@ private function removeLatestStableInstall()
             Filesystem::unlinkRecursive($installSubdirectory, true);
         }
 
+        if (file_exists($this->getBuildArchivePath())) {
+            Filesystem::unlinkRecursive($this->getBuildArchivePath(), true);
+        }
+
         $latestStableZip = $this->getArchiveDestPath();
         if (file_exists($latestStableZip)) {
             unlink($latestStableZip);
         }
     }
 
-    private function downloadAndUnzipLatestStable()
+    protected function downloadAndUnzipLatestStable()
```

Comment:
Probably not needed change of visibility?

Reply:
Probably not, Guess for a test file that doesn't bother much. In general I think `private` should only be used in case you don't want a method to be overwritten or accessible in an inherited class, which I don't think would be that case here.